newFile.py

```diff
@@ -0,0 +1,22 @@
+stages:
```

Comment:
Reviewing changes in **newFile.py**:

Clean and well-organized GitLab CI/CD pipeline script focusing on reviewing and improving PR's, good job on the environment variable management!